packages/forms-web-app/src/dynamic-forms/questions.js

```diff
@@ -44,7 +44,8 @@ exports.questions = {
 	}),
 	listedBuildingCheck: new BooleanQuestion({
 		title: 'Affects a listed building',
-		question: 'Do the plans affect the setting of a listed building or site?',
```

Comment:
there appear to have been some content changes to this question, which I've applied here